package/network/services/dropbear/Makefile

```diff
@@ -41,6 +57,10 @@ PKG_CONFIG_DEPENDS:= \
 
 include $(INCLUDE_DIR)/package.mk
 
+TARGET_CFLAGS += -DARGTYPE=3 -ffunction-sections -fdata-sections -flto=auto -ffat-lto-objects -Wl,-pie
+TARGET_CFLAGS += $(if $(filter -Os,$(TARGET_CFLAGS)),-DLTC_SMALL_CODE,-DDROPBEAR_SMALL_CODE=0)
+TARGET_LDFLAGS += -Wl,--gc-sections -flto=auto -ffat-lto-objects -pie
```

Comment:
Please do these optimizations in a separate  commit

Reply:
Do you really required -ffat-lto-objects? -fno-fat-lto-objects (which is the default) would be preferred if that builds and no static libraries are created for other packages.
Also, it's a compiler switch, you can remove that switch from the linker flags